qa/workunits/rados/test_rados_tool.sh

```diff
@@ -304,9 +314,22 @@ test_xattr() {
     rm $V1 $V2
     cleanup
 }
+test_rmobj() {
+    ceph osd pool create $p 1
+    ceph osd pool set-quota $p max_objects 1
+    V1=`mktemp fooattrXXXXXXX`
+    rados put $OBJ $V1 -p $p
+    sleep 30
```

Comment:
Do you mind replacing this with a loop that looks for the full flag on teh pool in the 'ceph osd dump' output?  Otherwise this will slow down the make check execution.